config/structs.go

```diff
@@ -36,6 +36,13 @@ type ListenConfig struct {
 	Port            int
 	Address         string
 	MetricsEndpoint string `hcl:"metrics_endpoint" yaml:"metrics_endpoint"`
+	TLS				TLSConfig
+}
+//TLSConfig is a struct for certificate files
+type TLSConfig struct {
+	Enable      bool
+	CertFile 	string
+	KeyFile	    string
```

Comment:
Can we add the respective `hcl:` and `yaml:` annotations for snake_cased keys? All other configuration properties use snake_case.